src/commands/helm-lint.yaml

```diff
@@ -2,7 +2,11 @@
 parameters:
   chart:
     type: "string"
+  ct_config:
+    description: Chart Testing Config file path
+    type: "string"
+    default: ""
 description: Lint Helm chart
 steps:
   - run: |
-      ct lint --validate-maintainers=false --charts="helm/<<parameters.chart>>"
+      ct lint <<# parameters.ct_config >> << parameters.ct_config >> <</ parameters.ct_config >> --validate-maintainers=false --charts="helm/<<parameters.chart>>"
```

Comment:
Is this `#` `/` a special syntax?

Reply:
It works as a conditional for booleans -> https://circleci.com/docs/2.0/reusing-config/#boolean

Since a string is false if it's empty, I presumed it would work but it doesn't :crying_cat_face: 

I'll update to use `when`

Reply:
Actually, it does work. I was missing an CLI argument in my config.